win32ss/gdi/ntgdi/freetype.c

```diff
@@ -88,7 +88,7 @@ static BOOL             g_RenderingEnabled = TRUE;
 #define ASSERT_FREETYPE_LOCK_NOT_HELD() \
     ASSERT(g_FreeTypeLock->Owner != KeGetCurrentThread())
 
-#define MAX_FONT_CACHE 256
+#define MAX_FONT_CACHE 400
```

Comment:
What does the 400 mean logically? Does it mean up to 400 fonts are stored, or does it mean anything else?
Why is that increase needed as part of the simplification?

LGTM otherwise, Please perform a testbot run and attach the results here before merging.

Reply:
Reverted.